Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+# What is an Inclusive Minute
+
+An inclusive minute (or inclusive moment) is an inclusivity practice designed for teams to easily add to existing meeting agendas.
+The practice is intentionally kept small, informal and uncontroversial to maximize the likelihood of adoption and uptake.
+The purpose is to raise awareness, inspire dialog, deeper thinking and/or inform future action.
+Although rather modest in its aims, especially when compared with the need it is intended to address, it is also a useful approach in helping to seed changes in culture.
+
+While a minute is never enough time, we aim to characterize a practice other teams can easily follow and adopt into their existing agendas.
+While we are all impacted by inclusion, sometimes in deeply personal ways, we aim for a focus that relates primarily to a team's existing work activities.
+While progress on inclusion is urgently needed, our modest aim here is simply to raise awareness and inspire further/deeper thought.
+
```

Comment:
Subheading here: "What does an inclusive minute look like?"

Reply:
I think the existing heading (well, *existing* since I moved old heading to title as you suggested in an earlier comment), what is an inclusive minute, captures ok